core/primitives/src/version.rs

```diff
@@ -61,3 +61,28 @@ impl ProtocolVersionRange {
         self.lower <= version && self.upper.map_or(true, |upper| version < upper)
     }
 }
+
+/// New Protocol features should go here. Nightly features are guarded by `#[cfg(feature = "nightly_protocol")]`.
```

Comment:
The features should be guarded by the dedicated compile-time feature flag (e.g. `protocol_feature_evm`) instead of `nightly_protocol`, so the transition from nightly to stable is just a change in the Cargo.toml (list the `protocol_feature_evm` in the `default` feature)